build-system/tasks/compile.js

```diff
@@ -351,100 +344,104 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
     }
     externs.push('build-system/amp.multipass.extern.js');
 
+    const platformOptions = {
```

Comment:
Nit: Add a comment

Reply:
Done in c14604d325f1c22fdea99bcf41028b2d8a76bd84